package.json

```diff
@@ -3,9 +3,10 @@
   "name": "apollo-client-error-template",
   "version": "1.0.0",
   "license": "MIT",
+  "main": "src/index.jsx",
```

Comment:
Does this need to be set for codesandbox?

Reply:
Yep! That or renaming `index.jsx` to `index.js`, which feels wrong because it has jsx elements